tests/test_2008-ak-type-layout.py

```diff
@@ -0,0 +1,16 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import re
+
+import awkward as ak
+
+
+def test_simple():
+    assert re.match(r"3 \* u?int\d+", str(ak.type([1, 2, 3])))
```

Comment:
This is overkill because the int type is not platform specific, as I had feared. (ArrayBuilder always returns int64)
```suggestion
    assert re.match(r"3 \* u?int\d+", str(ak.type([1, 2, 3])))
```